src/components/NewLocationPage/AboveTheFold/AboveTheFold.style.tsx

```diff
@@ -1,19 +1,30 @@
 import styled from 'styled-components';
 import { COLOR_MAP } from 'common/colors';
-import { maxContentWidth } from 'components/NewLocationPage/Shared/Shared.style';
 import {
   materialSMBreakpoint,
   mobileBreakpoint,
   countyMapToFixedBreakpoint,
 } from 'assets/theme/sizes';
 
 export const MainWrapper = styled.div`
+  padding: ${props => props.theme.spacingTheme.contentGutterMobile};
   background-color: ${COLOR_MAP.GREY_0};
-  padding: 2rem 1rem;
-  margin-bottom: 3rem;
 
   @media (min-width: ${mobileBreakpoint}) {
-    padding: 2rem;
+    padding: ${props => props.theme.spacingTheme.contentGutterDesktop};
+  }
+`;
+
+export const ContentContainer = styled.div`
+  margin: auto;
+  display: flex;
+  border: 1px solid red;
+  width: fit-content;
+
+  @media (min-width: ${countyMapToFixedBreakpoint}) {
+    width: 1220px; // plus 2 rem margin
+    width: ${props =>
+      props.theme.spacingTheme.locationPage.widthContentWithStickyMap};
```

Comment:
since we want the `position:fixed` county map to be **inside** of the centered content container, we have to hard code the width. otherwise, the centered div will only include the non-fixed items (ie. the above the fold grid)